### PR TITLE
Transpile multiple files

### PR DESCRIPTION
**Pull request by JorikSchellekens**
Naive way to compile any number of sol files. Handy for transpiling whole projects.

### CODE REVIEW

**Review by DomHenderson — COMMENTED** [review]
My only concern with this is that if you're trying to transpile just a few files then doing them one by one is fine, and if you're trying to transpile more than that then you really need some sort of pattern matching, so I'm not sure how much use this would get. Any thoughts?

**Comment by JorikSchellekens** [comment]
Pattern matching works out of the box with globbing

File: src/index.ts [hunk 1]
```diff
@@ -57,15 +57,20 @@ program
...
-    if (!isValidSolFile(file)) return;
-    try {
-      transpile(compileSolFile(file, options.warnings), options).map(([name, cairo, abi]) => {
-        outputResult(name, cairo, options, '.cairo', abi);
-      });
-    } catch (e) {
-      handleTranspilationError(e);
-    }
+  .action((files: string[], options: CliOptions) => {
+    if (files.some((file) => !isValidSolFile(file))) return;
```
Comment (swapnilraj) [inline src/index.ts:61]: Shouldn't we check if we every sol file is valid, if not we should print an error about the invalid ones

Comment (JorikSchellekens) [reply src/index.ts:61]: As in give errors for all failing ones instead of fast failing as we do here?

**Review by swapnilraj — APPROVED** [review]

**Pull request merged**